src/com/akshay/captain/jira.main.kts

```diff
@@ -191,39 +282,159 @@ fun <E> List<E>.ifNotEmpty(function: () -> Unit): List<E> {
     return this
 }
 
-fun createReleaseVersion() {
+/**
+ * Creates a release version in the Jira
+ */
+fun createReleaseVersion(appVersion: String) {
     // create your json here
-
+    val description =
+        "https://biowink.atlassian.net/wiki/spaces/HF/pages/289570878/Android+Releases#AndroidReleases-Android$appVersion.0"
```

Comment:
nit:  don't hardcode that `.0`, request the devs to pass the exact version name, like `70.0`

Reply:
Understood. Will do